src/app/register-form/register-form.component.ts

```diff
@@ -0,0 +1,147 @@
+import {Component, OnInit} from '@angular/core';
+import {Validators, FormControl, FormBuilder} from '@angular/forms';
+
+import {Application} from '../shared/application';
+import {ApplicationService} from '../services/application.service';
+
+@Component({
+  selector: 'app-register-form',
+  templateUrl: './register-form.component.html',
+  styleUrls: ['./register-form.component.scss']
+})
+export class RegisterFormComponent implements OnInit {
+  constructor(private applicationService: ApplicationService, private fb: FormBuilder) {
+  }
+
+  get name() {
+    return this.registerForm.get('name');
+  }
+
+  get surname() {
+    return this.registerForm.get('surname');
+  }
+
+  get email() {
+    return this.registerForm.get('email');
+  }
+
+  get tel() {
+    return this.registerForm.get('tel');
+  }
+
+  get education() {
+    return this.registerForm.get('name');
+  }
+
+  get answerFreeTimeActivity() {
+    return this.registerForm.get('answerFreeTimeActivity');
+  }
+
+  get answerThreePartAgreement() {
+    return this.registerForm.get('answerThreePartAgreement');
+  }
+
+  get answerAvailable14To18() {
+    return this.registerForm.get('answerAvailable14To18');
+  }
+
+  get answerMotivation() {
+    return this.registerForm.get('answerMotivation');
+  }
+
+  get answerExperience() {
+    return this.registerForm.get('answerExperience');
+  }
+
+  get answerInfoAboutAcademy() {
+    return this.registerForm.get('answerInfoAboutAcademy');
+  }
+
+  serverErrorMessage: string;
+
+
+  registerForm = this.fb.group({
+    name: ['',
+      [
+        Validators.required,
+        Validators.maxLength(256)
```

Comment:
Some of these validators are set both in the html and typescript - that doesn't need to be done. 
Best keep the validators in the typescript, then you can skip the attributes in the html.